src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -970,19 +982,37 @@ def set_vmss_diagnostics_extension(
         no_auto_upgrade=False):
     '''Enable diagnostics on a virtual machine scale set
     '''
+    from azure.mgmt.compute.models import UpgradeMode
```

Comment:
Should use the versioned SDK imports since vm module supports multiple API versions.